certified-connectors/SurveyMonkey/readme.md

```diff
@@ -0,0 +1,11 @@
+# SurveyMonkey
+SurveyMonkey is the world's leading provider of web-based survey solutions. These solutions are used by companies, organizations, and individuals to gather the insights they need to make more informed decisions.
+
```

Comment:
@depfeife-msft Please refer to the [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) for readme.md and add the required sections.

Reply:
@depfeife-msft The publisher section is required.